autoscheduler/scraper/models/grades.py

```diff
@@ -0,0 +1,75 @@
+""" Heavily based off of Good Bull Schedules """
+
+from typing import Dict
+from django.db import models
+
+class GradeManager(models.Manager):
+    """ Connects to the Grades models so we can call
+        Grades.object.instructor_performance
+    """
+
+    def instructor_performance(
+            self, dept: str, course_num: str, instructor: str
+    ) -> Dict[str, float]:

```

Comment:
Since the values are floats or ints, we can use `Union` from `typing` to show that it can be either:
https://stackoverflow.com/questions/33945261/how-to-specify-multiple-return-types-using-type-hints

Reply:
So would this look like `Dict[str, Union(int, float)]`?

Reply:
Correct